db_s3_backup.py

```diff
@@ -177,16 +177,16 @@ def delete_local_backups(dir_path, backup_prefix, backup_extension, verbose = Fa
 
 	# Loads configuration file from JSON
 	try:
-		f=open(args.config_file, 'r')
+		with open(args.config_file, 'r') as f:
+			try:
+				config=json.loads(f.read())
+			except Exception, e:
```

Comment:
Can you change this to `ValueError`? It's the right error for JSON parsing error.